feature-libs/user/profile/components/register/register.component.ts

```diff
@@ -60,6 +61,8 @@ export class RegisterComponent implements OnInit, OnDestroy {
         value: false,
         disabled: this.isConsentRequired(),
       }),
+      additionalConsents:
+        this.registerComponentService?.generateAdditionalConsentsFormControl(),
```

Comment:
When we implement [this comment](https://github.com/SAP/spartacus/pull/17497/files#r1233799700),
we should gracefully fallback, when the returned value is undefined:

```diff
       additionalConsents:
-        this.registerComponentService?.generateAdditionalConsentsFormControl(),
+        this.registerComponentService.generateAdditionalConsentsFormControl?.() ??
+        this.fb.array([]),

Reply:
done